nfv_networker_ovs_dpdk.md

```diff
@@ -0,0 +1,192 @@
+        rx_queue: 2
+        members:
+          -
+            type: interface
+            name: nic3
+```
+
+This example also assumes that the Networker nodes:
+
+- PXE and boot settings are already configured.
+- Are at least one baremetal compute with DPDK interfaces.
+
+Create an `OpenStackDataPlaneNodeSet` CR file,
+e.g. `dataplane_cr.yaml` to represent the EDPM nodes. See
+[dataplane_v1beta1_openstackdataplanenodeset.yaml](https://github.com/openstack-k8s-operators/openstack-operator/blob/main/config/samples/dataplane_v1beta1_openstackdataplanenodeset.yaml)
+for an example to modify as described in this document.
+
+Do not yet create the CR in OpenShift as the edits described in the
+next sections are required.
+
+The example
+[dataplane_v1beta1_openstackdataplanenodeset_ovs_dpdk.yaml](https://github.com/openstack-k8s-operators/openstack-operator/tree/main/config/samples/dataplane/ovs_dpdk)
+has OVS DPDK interfaces network configuration and required edpm ansible parameters.
+
+Modify your `OpenStackDataPlaneNodeSet` CR to set
+[edpm-ansible](https://github.com/openstack-k8s-operators/edpm-ansible)
+variables so that the
+[edpm_network_config role](https://github.com/openstack-k8s-operators/edpm-ansible/blob/main/roles/edpm_network_config/)
+will configure networks with ovs dpdk interfaces.
+
+
+`OpenStackDataPlaneNodeSet`
+[EDPM services list](https://openstack-k8s-operators.github.io/openstack-operator/dataplane/#_composable_services)
+
+```yaml
+apiVersion: dataplane.openstack.org/v1beta1
+kind: OpenStackDataPlaneNodeSet
+spec:
+  ...
+  roles:
+    edpm-networker:
+      ...
+      services:
+        - bootstrap
+        - download-cache
+        - reboot-os
+        - configure-ovs-dpdk
+        - configure-network
+        - validate-network
+        - install-os
+        - configure-os
+        - ssh-known-hosts
+        - run-os
+        - install-certs
+        - ovn
+        - neutron-ovn
+        - neutron-metadata
+```
+
+### Create the OpenStackDataPlaneNodeSet
+
+Create the CR from your directory based on the example
+[dataplane_v1beta1_openstackdataplanenodeset_ovs_dpdk](https://github.com/openstack-k8s-operators/openstack-operator/tree/main/config/samples/dataplane/ovs_dpdk)
+with the changes described in the previous section.
+```
+oc kustomize --load-restrictor LoadRestrictionsNone openstack-operator/config/samples/dataplane/ovs_dpdk > dataplane_cr.yaml
+```
+
+### Create a OpenStackDataPlaneDeployment
+
+Creating an `OpenStackDataPlaneDeployment` will trigger Ansible jobs
+to configure an Networker. Which Ansible roles are run depends on the
+`services` list.
+
+Each `OpenStackDataPlaneDeployment` can have its own
+`servicesOverride` list which will redefine the list
+of services of an `OpenStackDataPlaneNodeSet` for a
+deployment.
+
+The example
+[dataplane_v1beta1_openstackdataplanedeployment_ovs_dpdk](https://github.com/openstack-k8s-operators/openstack-operator/tree/main/config/samples/dataplane/ovs_dpdk)
```

Comment:
This seems to be for the `OpenStackDataPlaneNodeSet`, not for the `OpenStackDataPlaneDeployment`, as least by looking at the `kustomization.yaml` and `values.yaml` listed in that directory